pkg/local_object_storage/shard/head.go

```diff
@@ -46,26 +46,25 @@ func (r HeadRes) Object() *objectSDK.Object {
 // Returns an error of type apistatus.ObjectAlreadyRemoved if the requested object has been marked as removed in shard.
 // Returns the object.ErrObjectIsExpired if the object is presented but already expired.
 func (s *Shard) Head(prm HeadPrm) (HeadRes, error) {
-	// object can be saved in write-cache (if enabled) or in metabase
-
-	if s.hasWriteCache() {
-		// try to read header from write-cache
-		header, err := s.writeCache.Head(prm.addr)
-		if err == nil {
-			return HeadRes{
-				obj: header,
-			}, nil
-		} else if !IsErrNotFound(err) {
-			// in this case we think that object is presented in write-cache, but corrupted
-			return HeadRes{}, fmt.Errorf("could not read header from write-cache: %w", err)
-		}
-
-		// otherwise object seems to be flushed to metabase
-	}
-
 	var obj *objectSDK.Object
 	var err error
+
 	if s.GetMode().NoMetabase() {
+		if s.hasWriteCache() {
```

Comment:
It make no sense, in degraded mode writecache is unavailable too. I think we can just drop this.

Reply:
dropped

> in degraded mode writecache is unavailable too

made me recheck that fact. isnt obvious, mb that a problem